Services/DeviceClient.cs

```diff
@@ -195,6 +200,12 @@ public async Task SendMessageAsync(string message, DeviceModel.DeviceModelMessag
         /// </summary>
         public async Task UpdatePropertiesAsync(ISmartDictionary properties)
         {
+            if (!this.deviceTwinEnabled)
```

Comment:
q: Do we need to initialize DeviceProperties actor if these global flags are not enabled ? Instead of adding a check here, can we add a check in simulation manager to skip creating device prop actors ?

Reply:
this code will never execute because the thread for properties is disabled. I added the code just as an extra protection in case one still wants to have the thread running to stress CPU and memory.